runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -355,11 +403,12 @@ class LayoutFFITypeHelpers
 	/* @brief Encode a primitive signature string to a predefined type which
 	 * is set in a struct representing the signature in the upcall.
 	 *
-	 * @param cSignature[in] A pointer to a preprocessed signature string
+	 * @param cSignature[in/out] A pointer to a preprocessed signature string, which always points
```

Comment:
This is not an "out" parameter, so there's nothing to describe for callers.

Reply:
Removed.